packages/paste-website/src/components/SlantedBackgroundGradient.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import {BackgroundColorOptions} from '@twilio-paste/style-props';
+import {useTheme} from '@twilio-paste/theme';
+import {useWindowSize} from '../hooks/useWindowSize';
+
+const SKEW_ANGLE = -0.191986; // radians. -11deg
```

Comment:
I think the angle here is a wee bit too steep, maybe by like 2 degrees 
![image](https://user-images.githubusercontent.com/368249/100125448-a7e4d880-2e31-11eb-93fb-935c61227153.png)

Reply:
Changed that to 9 degrees.